spec/lib/sarif/base_sarif_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../../spec_helper.rb'
+
+describe Sarif::BaseSarif do
```

Comment:
One thought as you layout all of the specs.  It would be really nice to include validation against the SARIF json schema in the specs.  That way we know as we build out the SARIF that it conforms to the spec.

See https://github.com/ruby-json-schema/json-schema

We could add the schema as a fixture here and validate against it.  That will ensure this first pass is valid and as you stand up scanners we stay compliant.  

```
JSON::Validator.validate(schema, your_generated_sarif_here)
```